emperor/support_files/js/color-view-controller.js

```diff
@@ -56,62 +55,108 @@ define([
     var SLICK_WIDTH = 25, scope = this;
     var name, value, colorItem;
 
+    // Create scale div and checkbox for whether using scalar data or not
+    this.$scaleDiv = $('<div></div>')
+    this.$colorScale = $("<svg width='90%' height='40'></svg>");
+    this.$scaleDiv.append(this.$colorScale);
+    this.$scaled = $("<input type='checkbox' id='scaled'>");
```

Comment:
As far as I can tell in the codebase (and @ElDeveloper correct me if I'm wrong) nothing is done through IDs directly, but instead using the objects attached to the controllers, so this is probably not needed.